### PR TITLE
Multiple Library Updates

### PR DESCRIPTION
**Pull request by WoogieNoogie**
Added Mixpanel and RxLifecycle rules.
Made modifications to the RxJava, RxAndroid, and SugarORM rules.

### CODE REVIEW

File: library/src/main/res/values/library_mixpanel_strings.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <string name="define_mixpanelandroid"></string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_mixpanel_strings.xml]: should be define_int_* as like here:
https://github.com/WoogieNoogie/AboutLibraries/blob/19167939b4ffccb09ada141d1dd46904e96d9d53/library/src/main/res/values/library_sugarorm_strings.xml#L3

File: library/src/main/res/values/library_rxlifecycle_strings.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <string name="define_rxlifecycle"></string>
```
Comment (mikepenz) [inline library/src/main/res/values/library_rxlifecycle_strings.xml]: should be define_int_* as like here:
https://github.com/WoogieNoogie/AboutLibraries/blob/19167939b4ffccb09ada141d1dd46904e96d9d53/library/src/main/res/values/library_sugarorm_strings.xml#L3

**Comment by mikepenz** [comment]
@WoogieNoogie which odd issues are you talking about?

**Comment by WoogieNoogie** [comment]
Mainly, I've included SugarORM and RxJava in my project, but neither are appearing in the about page. Hopefully the changes I made have fixed this, but at the very least, these two new libraries can be included. I've made the fixes from your comments above.

**Comment by mikepenz** [comment]
SugarORM is not showing up because it has a wrong classPath here:
https://github.com/mikepenz/AboutLibraries/pull/221/files#diff-2e6006abc2fa8c46012f43afc41c259aR16

Didn't see it before. 

The problem is that I have to try to init the class to find out if a specific library is included. So therefor I introduced the `classPath`. It may has a bit a wrong name. 

As an alternative the files can be included in the libs themself just having the `define_` prefix. This will tell the aboutlibs that the library is used and that the resource comes from the library and not from the AboutLibraries. 

So in case of SugarOrm it should be like:
https://github.com/mikepenz/AboutLibraries/pull/221/files#diff-2e6006abc2fa8c46012f43afc41c259aR16
`com.orm.SugarApp`

**Comment by WoogieNoogie** [comment]
Okay, maybe I didn't 100% understand how this library pulled in the other projects. It can't just use the package path, it needs to be a full path to a class that's used in the project - is that right?

**Comment by mikepenz** [comment]
Yes correctly. There is no way I have found which allows me to check for a specific package to exist

**Comment by WoogieNoogie** [comment]
Okay great, sorry for the confusion. Let me fix things up a bit and I'll comment when things are looking better.

**Comment by mikepenz** [comment]
@WoogieNoogie sieht gut aus.

**Comment by mikepenz** [comment]
@WoogieNoogie Oh sorry that was the wrong language. Looks fine ;)

**Pull request merged**